FEC-11335 Send Kava Impression on change media

GouravSna opened the pull request.

GouravSna committed:
- Fire Kava impression

GouravSna committed:
- Checking combined Plugin config to know if kava is present

giladna commented on tvplayer/src/main/java/com/kaltura/tvplayer/KalturaPlayer.java:
```diff
@@ -410,16 +408,16 @@ private void updatePlayerSettings() {
     private PKPluginConfigs setupPluginsConfiguration() {
         PKPluginConfigs pluginConfigs = initOptions.pluginConfigs;
-        PKPluginConfigs combinedPluginConfigs = new PKPluginConfigs();
+        this.combinedPluginConfigs = new PKPluginConfigs();
```
@GouravSna  no need in this

GouravSna replied:
Done

giladna commented on tvplayer/src/main/java/com/kaltura/tvplayer/KalturaPlayer.java:
```diff
@@ -1210,6 +1213,38 @@ private PhoenixAnalyticsConfig getPhoenixAnalyticsConfig() {
         return new PhoenixAnalyticsConfig(getPartnerId(), getServerUrl(), getKS(), Consts.DEFAULT_ANALYTICS_TIMER_INTERVAL_HIGH_SEC);
     }
 
+    private void sendKavaImpression(String entryId) {
```
lets review the logic to call this method

GouravSna committed:
- Code refactor for firing the KavaEvent
- For Basic Player, firing Impression on player creation with default values
- Firing Play_Request for each media playback

GouravSna committed:
- Added sessionId for analytics calls

GouravSna committed:
- Travis build

GouravSna committed:
- Travis build

GouravSna committed:
- Reverted the wrong code

GouravSna committed:
- Travis build

GouravSna committed:
- Travis build
